### DISCUSSION TITLE
Few Windows bugs/problems

### DISCUSSION THREAD

**Discussion by ghost** [xournalpp/xournalpp#3592] [General]
Hello,

This app is awesome, but there are a few consistent problems I have run into.

1. After you move on from the first page in the journal, the side display of pages will not update properly and display your page as a blank white page. The app will not allow you to scroll nor jump to the first page and the side display will not allow you to click on the page. The only way to access that page again is to save and reset your app.

Before reset:
![Screenshot_1](https://user-images.githubusercontent.com/64975010/142721692-1e5729b9-d205-4a76-af7b-19b40c98f440.jpg)

After reset:
![Screenshot_2](https://user-images.githubusercontent.com/64975010/142721695-6cc5f8c9-a0b4-4c83-9a97-b59905984ff3.jpg)

2. When typing in textboxes random letters or words will get cut off on the top and resizing the textbox will either not fix the cut off portion or add new cut off portions.

Cutoff 'f' in textbox:
![Screenshot_3](https://user-images.githubusercontent.com/64975010/142721709-27725beb-0746-4e02-89b4-e6205dad9db1.jpg)

3. When working with custom backgrounds from PDFs, it will only apply the background to one page at a time, the only way to consistently get a repeating background is to duplicate the above page and erase all the contents. Using the journals 'configure page template' setting will not change anything and your backgrounds will still always be blank white. When trying to use the apply background to all pages option with a PDF background, it will instantly freeze and/or crash the entire app.

4. When working with large selected areas, the entire app will move with the selection meaning if you move the selection down even just a millimeter it will jump to a random page at the bottom of your journal and it makes it very difficult to move the selected area to the desired location.

5. When drawing lines or arrows, stretching out the shape will show a perfectly straight line/arrow but once you release the mouse button it will be a crooked shape despite the preview. It would be nice if a key bind - like shift on most apps - were added so that the lines would only be able to be drawn at 0, 45, and 90 degree angles.

**Comment by LittleHuba** [comment]
1-2 should already be fixed in the latest nightly - #3534 

Sorry Nr. 3 is something different (maybe?).

**Comment by rolandlo** [comment]
Nr. 4 was fixed in the latest nightly by https://github.com/xournalpp/xournalpp/pull/2893

@mattbcraft Please add some images for Nr. 5, I don't get the problem and don't know in what direction you scale.

**Reply by ghost** [reply]
> Nr. 4 was fixed in the latest nightly by #2893
> 
> @mattbcraft Please add some images for Nr. 5, I don't get the problem and don't know in what direction you scale.

Arrow going down is shown as straight while I draw it (mouse button held down)
![image](https://user-images.githubusercontent.com/64975010/143613636-94d5bfad-988f-4478-8bfb-871eb59bb29c.png)

Arrow ends up being crooked despite straight preview (mouse button released)
![image](https://user-images.githubusercontent.com/64975010/143613725-db59a898-c033-42fe-bb3e-e196e74a19a4.png)
 
A keybind like holding shift to ensure straight drawings would be helpful - I find this error coming up randomly while I work and I have to try and draw a straight arrow for a few minutes before it finally actually ends up straight
